$MirOS: ports/www/firesomething/patches/patch-nsprpub_pr_src_misc_prdtoa_c,v 1.2 2008/12/05 16:26:11 bsiegert Exp $
$OpenBSD: patch-nsprpub_pr_src_misc_prdtoa_c,v 1.1 2005/11/03 00:59:43 pvalchev Exp $
--- nsprpub/pr/src/misc/prdtoa.c.orig	Fri Mar 20 03:32:47 2009
+++ nsprpub/pr/src/misc/prdtoa.c	Mon Aug  3 18:41:06 2009
@@ -840,7 +840,7 @@ mult
 	wc = wa + wb;
 	if (wc > a->maxwds)
 		k++;
-	c = Balloc(k);
+	c = Balloc(k+2);
 	for(x = c->x, xa = x + wc; x < xa; x++)
 		*x = 0;
 	xa = a->x;
